app/views/users/feed.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="row mb-4">
+  <div class="col md-6 offset-3">
```

Comment:
nitpik: There's a small typo in your grid class. Instead of class="col md-6 offset-3", it should be class="col-md-6 offset-md-3". The correct syntax will ensure the column is centered and has the desired width on medium-sized devices and above.